deployment/constructs/lambda.ts

```diff
@@ -8,10 +8,10 @@ import { Duration, RemovalPolicy, aws_s3 as s3 } from "aws-cdk-lib";
 import { Runtime } from "aws-cdk-lib/aws-lambda";
 import { PolicyStatement } from "aws-cdk-lib/aws-iam";
 import * as apigateway from "aws-cdk-lib/aws-apigateway";
-import { isLocalStack } from "../local/util";
+import { isLocalStack } from "../local/util.js";
```

Comment:
nit: since we're setting this up for the first time could we not configure the package and tsconfig so we don't need `.js` extensions on imports?

Reply:
I wanted to do this, but I couldn't find a way to without configuring tsconfig.json with settings I don't think we should have.  `moduleResolution: bundler` would alllow us to do that, but I don't think that's right.